xbmc/storage/MediaManager.h

```diff
@@ -118,7 +119,7 @@ class CMediaManager : public IStorageEventsCallback, public IJobCallback
   std::string m_strFirstAvailDrive;
 
 private:
-  IStorageProvider *m_platformStorage;
+  std::unique_ptr<IStorageProvider> m_platformStorage{nullptr};
```

Comment:
`{nullptr}` is not necessary here, as default smart ptr constructor initializes it to owns nothing.

Is similar to initialize std::string or std::vector.

Reply:
yeah I know, though that initializing it explicitly would make it more clear. Adapted, thanks.